lib/praxis/request.rb

```diff
@@ -85,85 +85,83 @@ def self.path_version_prefix
     # DEPRECATED: remove with ResourceDefinition.version using: :path
     PATH_VERSION_MATCHER = %r{^#{self.path_version_prefix}(?<version>[^\/]+)\/}.freeze
 
-    def path_version_matcher
```

Comment:
something's gone wrong with the formatting of this file....not even sure there are any real changes?